testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -398,7 +398,7 @@ class TestGuessBonds(object):
      - fail properly if not
      - work again if vdwradii are passed.
     """
-    @pytest.fixture()
+    @pytest.fixture(scope='module')
```

Comment:
why is scope changing?

Reply:
When we run the tests we don't need to call `vdw` each time we need it. We can only call it once when first called as it's values does not change across the tests. Although it does a simple task, I guess it's better to be done once, rather than repeatedly for each test function.

Reply:
Makes sense to me to make "read only" fixtures to be module-scoped  — even though the speed-up is almost certainly negligible for something simple like a dict.